src/cms/templates/pages/page_tree_node.html

```diff
@@ -33,6 +33,13 @@
             {{ page_translation.get_status_display }}
         </a>
     </td>
+    <td class="whitespace-nowrap">
+        <a href="{% url 'edit_page' page_id=page.id region_slug=region.slug language_slug=language.slug %}" class="block py-3 px-2 text-gray-800">
+            {% for tag in page_translation.tags %}
+                <span class="tag bg-orange-400 text-white rounded px-2 py-1">{{ tag }}</span>
```

Comment:
```suggestion
                <span class="bg-orange-400 text-white rounded px-2 py-1">{{ tag }}</span>
```
The `tag` class is no longer needed, but apart from that I can't find anything to niggle about :wink: